apps/app-article/src/App.js

```diff
@@ -460,14 +459,22 @@ if (window.ksfDfp) {
 
     onUserFetchSuccess(user) {
         //Cookie will expire after 7 days 
-        Cookies.set('LoginStatus', true, { expires: 7 });
+        Cookies.set('LoginStatus', true, { expires: 365 });
+        Cookies.set('token', localStorage.getItem('token'), { expires: 365 });
+        Cookies.set('uuid', localStorage.getItem('uuid'), { expires: 365 });
         //To get User data from Android side 
-        Cookies.set('currentUser', JSON.stringify({ firstName: user.firstName, lastName: user.lastName, email: user.email, token: localStorage.getItem('token'), uuid: localStorage.getItem('uuid') }));
+        Cookies.set('currentUser', JSON.stringify({ firstName: user.firstName, lastName: user.lastName, email: user.email, token: localStorage.getItem('token'), uuid: localStorage.getItem('uuid') }, {expires: 365}));
         localStorage.setItem("currentUser", JSON.stringify(user));
         this.setState({user: user});
-        this.fetchArticleFromApi(getUrlParam().has('uuid')?getUrlParam().get('uuid'):"");
-        //Call Android bridge 
-        Android.isLoggedIn();        
+        if(getUrlParam().has('uuid') && getUrlParam().has('uuid') != ""){
+            this.fetchArticleFromApi(getUrlParam().has('uuid')?getUrlParam().get('uuid'):"");
```

Comment:
Could we assign `getUrlParam().get('uuid')` to a variable and work with that instead of checking the url params multiple times? 🙂 Something like:
```
const articleUuid = getUrlParam().get('uuid');
if (articleUuid) { 
  this.fetchArticleFromApi(articleUuid);
}
```
This would make the code a bit easier for the reader.

Reply:
yes this looks much cleaner thnx! already commited the change!